packages/material-ui-lab/src/AlertTitle/AlertTitle.d.ts

```diff
@@ -6,4 +6,14 @@ export interface AlertTitleProps
 
 export type AlertTitleClassKey = 'root';
 
+/**
+ *
+ *
+ * Demos:
+ * -
```

Comment:
Empty

Reply:
Just like https://material-ui.com/api/alert-title/. Shouldn't we force that every public component has a demo?

Reply:
Right, it seems that we forgot to mention this component, the fix would be:

```diff
diff --git a/docs/src/pages/components/alert/alert.md b/docs/src/pages/components/alert/alert.md
index 6c726cd11..c207f8537 100644
--- a/docs/src/pages/components/alert/alert.md
+++ b/docs/src/pages/components/alert/alert.md
@@ -1,6 +1,6 @@
 ---
 title: Alert React component
-components: Alert
+components: Alert, AlertTitle
 ---

 # Alert
```

Reply:
> Shouldn't we force that every public component has a demo?

No objection to force that

Reply:
Yeah let's do this in a follow-up. RootRef also has no demos and I didn't check if this is legit or not.

Reply:
Oops, too late, I have pushed a commit with AlertTitle. For RootRef, assuming we remove it in v5, I wouldn't worry about it :)